### PR TITLE
HOLD FOR RELEASE - 60674: Registry Updates

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/60674/vendor-team-redesign-of-registry-ui

Previews:

- https://deploy-preview-707--replicated-docs.netlify.app/vendor/packaging-private-images#configure-access-to-an-external-registry
- https://deploy-preview-707--replicated-docs.netlify.app/vendor/packaging-private-images#configure-access-with-the-vendor-portal
- https://deploy-preview-707--replicated-docs.netlify.app/vendor/packaging-private-images#configure-access-with-the-cli

### CODE REVIEW

File: docs/vendor/packaging-private-images.md [hunk 1]
```diff
@@ -37,67 +37,183 @@ You can provide the credentials for an external registry in the vendor portal to
 grant the app manager proxy access to the private application images in the
 registry.
 
-All applications in your vendor portal Team have access to the external registry
+All of the applications in your vendor portal Team have access to the external registry
 that you add. This means that you can use the images in the external registry across
-multiple apps in the Team.
+multiple applications in the Team.
+
+The External Registries pane on the Images page displays all of the external registry connections and displays any errors in the registry log files. You can also test your connection to the registries or remove registries from this page at any time.
```
Comment (alicenstar) [inline docs/vendor/packaging-private-images.md:44]: The wording around "test your connection" could maybe be changed. The 'test' button actually tests an image pull from your specified registry, if that helps.

**Comment by jonquil2002** [comment]
@ajp-io  Do you want to do a PM review on this? Not sure if you're covering this team's features or not right now. Star reviewed it.

**Review by alicenstar — APPROVED** [review]

**Comment by ajp-io** [comment]
@marccampbell is serving as the PM for the vendor portal team right now.

**Review by paigecalvert — APPROVED** [review]

File: docs/vendor/packaging-private-images.md [hunk 2]
```diff
@@ -37,67 +37,183 @@ You can provide the credentials for an external registry in the vendor portal to
...
+
+  **quay.io:**
+  <table>
+    <tr>
+      <th width="30%">Field</th>
+      <th width="70%">Description</th>
+    </tr>
+    <tr>
+      <td>Endpoint</td>
+      <td>Enter the endpoint, such as <code>gcr.io</code>.</td>
```
Comment (paigecalvert) [inline docs/vendor/packaging-private-images.md:143]: Just noticed a typo here: I think this example should be `quay.io` (or something) and not `gcr.io`. Probably just a copy/paste error

Comment (jonquil2002) [reply docs/vendor/packaging-private-images.md]: @paigecalvert Fixed, please reapprove. Thanks.

**Review by paigecalvert — APPROVED** [review]

**Comment by jonquil2002** [comment]
@alicenstar  Is this registry change still behind a feature flag? I noticed that PM hasn't merged this doc yet and am wondering about the status.

**Comment by alicenstar** [comment]
@jonquil2002 This *is* still behind a feature flag. With the instance details reporting getting prioritized on our team, we haven't had time to revisit this yet. I can try to bring it up next week.

**Comment by jonquil2002** [comment]
Closing this PR. It is stale. If the feature is ever released, we will need to rewrite it because of the heavy content and TOC changes that are being implemented for the Builders Plan.

**Pull request closed without merging**